component/src/test/java/org/wso2/extension/siddhi/io/websocket/util/WebSocketEndpoint.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+package org.wso2.extension.siddhi.io.websocket.util;
+
+import java.nio.ByteBuffer;
+import javax.websocket.OnMessage;
+import javax.websocket.Session;
+import javax.websocket.server.ServerEndpoint;
+
+@ServerEndpoint(value = "/abc", subprotocols = {"chat", "superchat"})
+public class WebSocketEndpoint {
+
+    @OnMessage
+    public void onTextMessage(Session session, String message) {
+        for (Session s : session.getOpenSessions()) {
+            s.getAsyncRemote().sendText(message);
+        }
+    }
+
+    @OnMessage
+    public void onBinaryMessage(Session session, byte[] message) {
+        for (Session s : session.getOpenSessions()) {
```

Comment:
Change variable name `s` to `session` to improve readability.

Reply:
fixed in 9fea7d3